### ISSUE TITLE
Add skip to main link back to new homepage

### ISSUE THREAD

**Issue by brentryanjohnson** [ONRR/doi-extractives-data#4522]
The new homepage work eliminated the `id` necessary for the skip to main content link to work. We need to add it back.

**Comment by brentryanjohnson** [comment]
I added a skip to main content keyboard link. Now I need to audit pages for `<main>` elements, and add the link `id` to pages in which it's missing.
